user_manual/changes/S916-015.yaml

```diff
@@ -0,0 +1,6 @@
+type: new-feature
+title: Add property ``EntryDecl.p_accept_stmts``
+description: |
+    This new property returns an array of accept statements calling
```

Comment:
`calling` is not really the right word here. What about `When invoked on a task entry declaration, this property returns the array of all its corresponding accept statements.`?

Reply:
Sounds good, thanks.